packaging/arch/regenerate_images

```diff
@@ -3,15 +3,25 @@
 # find out all installed kernels
 mapfile -d '' kernels < <(find /usr/lib/modules -maxdepth 1 -type d ! -name "modules" -print0)
 
+# paths to boot partition for the initramfs image and kernel
+boot_paths=("/efi" "/boot")
+
 for kernel in "${kernels[@]}"; do
   if ! pacman -Qqo "${kernel}/pkgbase" > /dev/null 2>&1; then
     # if pkgbase does not belong to any package then skip this kernel
     continue
   fi
   read -r pkgbase < "${kernel}/pkgbase"
 
-  booster build --force --kernel-version ${kernel##/usr/lib/modules/} /boot/booster-${pkgbase}.img &
-  install -Dm644 "${kernel}/vmlinuz" "/boot/vmlinuz-${pkgbase}"
+  # build and install the image and kernel at one boot path
+  for path in "${boot_paths[@]}"; do
+    if [ -d "${path}" ] && [ -w "${path}" ]; then
+      booster build --force --kernel-version "${kernel##/usr/lib/modules/}" "$path"/booster-"${pkgbase}".img &
+      install -Dm644 "${kernel}/vmlinuz" "${path}/vmlinuz-${pkgbase}"
+
+      break
```

Comment:
I believe this is wrong and we shouldn't `break`. In the case that `/efi` exists and is empty, and the boot path of the user is **actually** `/boot`, then neither the image and the kernel will not be installed.

Besides being lazy and installing them at all paths, is there any way to determine the users' boot path?

Reply:
You can get the ESP or BOOT paths with the `bootctl` command, something like this:
```Shell
esp_path="$(bootctl -p)"
boot_path="$(bootctl -x)"
```

If the `XBOOTLDR` partition is used, esp_path and boot_path should be different

Reply:
Thanks but that won't work in all of the cases. `bootctl` is dependent and part of the systemd project.